backstage/src/ruleset/ops.ts

```diff
@@ -1,2 +1,2 @@
 export type LogicOp = 'and' | 'or' | 'not'
-export type PropertyOp = 'add' | 'sub' | 'set' | 'mul'
+export type PropertyOp = 'add' | 'sub' | 'set_min' | 'set_max'
```

Comment:
And we need a `set_incr` event for increment values every turn (i.e. "talent" values). Also, this needs to be handled properly in `nextTurn` function.